java/dagger/example/android/espresso/stubbed/StubNetworkingModule.java

```diff
@@ -0,0 +1,18 @@
+package dagger.example.android.espresso.stubbed;
+
+import dagger.Module;
+import dagger.Provides;
+import dagger.example.android.espresso.main.ApiClient;
+
+@Module
+public class StubNetworkingModule {
```

Comment:
Why redeclare module instead of extending and overriding only those `@Provides` methods that you need?

Reply:
bump. is this going to get merged?